editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -840,23 +844,32 @@ const RowForObjectControl = React.memo((props: RowForObjectControlProps) => {
             style={{ flexGrow: 1, justifyContent: 'space-between', paddingRight: 10 }}
             ref={dataPickerButtonData.setReferenceElement}
           >
-            <SimpleFlexRow style={{ flexGrow: 1, paddingRight: 8 }} onClick={handleOnClick}>
+            <SimpleFlexRow
+              style={{ flexGrow: 1, paddingRight: 8, justifyContent: 'space-between' }}
+              onClick={handleOnClick}
+            >
               <PropertyLabel
                 target={[propPath]}
                 style={{
-                  textTransform: 'capitalize',
+                  ...objectPropertyLabelStyle,
                   paddingLeft: indentation,
-                  display: 'flex',
-                  alignItems: 'center',
-                  height: 34,
-                  fontWeight: 500,
-                  gap: 4,
                   cursor: props.disableToggling ? 'default' : 'pointer',
                 }}
               >
                 {title}
                 {unless(props.disableToggling, <ObjectIndicator open={open} />)}
               </PropertyLabel>
+              <div onClick={stopPropagation}>
+                <ControlForProp
```

Comment:
this is the interesting change, RowForObjectControl renders ControlForProp which will render a Cartouche (if applicable) or return `null` if no cartouche was needed